include/revng/EarlyFunctionAnalysis/CallEdge.h

```diff
@@ -46,15 +46,18 @@ class efa::CallEdge : public efa::generated::CallEdge {
     AssociatedType = FunctionEdgeType::FunctionCall;
 
 public:
+  using generated::CallEdge::CallEdge;
   CallEdge() : efa::generated::CallEdge() { Type() = AssociatedType; }
-
-  CallEdge(MetaAddress Destination, FunctionEdgeType::Values Type) :
-    efa::generated::CallEdge(Destination, Type) {}
+  CallEdge(MetaAddress Destination, efa::FunctionEdgeType::Values Type) :
+    efa::generated::CallEdge() {
+    this->Destination() = Destination;
+    this->Type() = Type;
+  }
```

Comment:
Why?
What's going on in this commit?

Reply:
@drblallo, for some reason, possibly due to some commit in `develop`, EFA CallGraph & yield CrossRelation got desync'd. Sync'ing back is the purpose of this commit.

Reply:
i can't offer a explanation better than antonion's one